engine/src/main/java/com/datatorrent/stram/engine/AbstractReservoir.java

```diff
@@ -47,7 +49,7 @@
 {
   private static final Logger logger = LoggerFactory.getLogger(AbstractReservoir.class);
   static final String reservoirClassNameProperty = "com.datatorrent.stram.engine.Reservoir";
-  private static final int USE_SPSC_CAPACITY = 8 * 1024;
+  private static final int USE_SPSC_CAPACITY = 64 * 1024;
```

Comment:
You mean it exceeds SpscArrayQueueReservoir throughput for capacity < 64K? Would be great to see the numbers you got added to the JIRA.